tests/IBTK/curl_01_3d.output

```diff
@@ -0,0 +1 @@
+|e|_oo = 2.14735
```

Comment:
The large error is because of the coarse-fine interface treatment, which is using `CONSERVATIVE_LINEAR_REFINE` to fill ghost values --- so we only expect (and realize) first order accuracy.